tests/core/test_policies.py

```diff
@@ -6,7 +6,7 @@
 import numpy as np
```

Comment:
thanks for cleaning this up! There are two more `TODO`s in the code which I believe can be removed, right? (Line 353 and 310)